### ISSUE TITLE
angular-seed compatible?

### ISSUE THREAD

**Issue by tomlobato** [neroniaky/angular2-token#108]
I`m having a hard time to make it work with https://github.com/mgechev/angular-seed
Anyone did it successfully already?

**Comment by tomlobato** [comment]
Well, no way to make this package work in angular-seed.
I\`ve ended up copying `src/angular2-token.service.ts` and `src/angular2-token.model.ts` into my project folder `src/client/shared/angular2-token/` and importing it directly by relative local path.

src/client/shared/shared.module.ts:
```ts
import { Angular2TokenService } from './angular2-token/angular2-token.service';
export class SharedModule {
    providers: [
        Angular2TokenService
```

src/client/app.component.ts:
```ts
import { Angular2TokenService } from './shared/angular2-token/angular2-token.service';
export class AppComponent {
  constructor(private angular2TokenService: Angular2TokenService) {
    this.angular2TokenService.init({
      apiPath: 'http://localhost:3000'
    });
```

src/client/shared/toolbar/toolbar.component.ts:

```ts
import { Angular2TokenService } from '../angular2-token/angular2-token.service';
export class ToolbarComponent {
    constructor(protected angular2TokenService: Angular2TokenService){}
    logout() {    
      this.angular2TokenService.signOut()
        .subscribe(
          resp => {
            this.router.navigate(['/login']);
          },
          error => {
            console.error(error);
          }
    );
  }
```

**Comment by neroniaky** [comment]
I guess this might be related to #103. I'm on it, but couldn't get the it to work yet.

**Comment by neroniaky** [comment]
I just published version 0.2.0-beta.6, which should work a lot better, because it uses the compiled .js files. Could you give it another try?
